applications/Crosschainstorage.md

```diff
@@ -0,0 +1,138 @@
+# Crust Grant Proposal
+
+* Project Name: CCS - Cross Chain Storage Platform
+* Team Name: Zhu, Wenfeng (朱文峰); Gao, Le (高乐); Wang, Ruijun (王瑞俊); Sang, Junjun (桑俊俊); Dong, Yuansheng (董远胜); Liu, Qi (刘琦)
+* Payment Address: 0x17389d755cb720e5e7e12d0fb926c0d53a2846d6 (USDT)
+* Receivable Token: USDT – ERC20
+
+
+### Overview
+
+* A project name: CCS - Cross Chain Storage Platform
+* A brief description of your project: In the ear of data explosion and related storage request, we want to create a decentralized, highly motivated storage platform. We will provide customer (individual and business) with a more secure, privacy-protected and cost-effective data storage service. 
+* CCS will build multiple connection adapter incentive layers; Crust will be the pilot and main storage channel layer for CCS.
+* Our team sees an opportunity to consolidate multiple decentralized storage providers to build a platform like “去哪儿” in air-tickets booking network, which provide business clients with most optimized data / files storage solutions.
+
+
+### Project Details 
+
+* Mockups/designs of any UI components
+![image](https://user-images.githubusercontent.com/47582549/124770266-c89b1200-df6c-11eb-8737-0cafd60aba4a.png)
+![image](https://user-images.githubusercontent.com/47582549/124770364-da7cb500-df6c-11eb-96e7-b0f025cd60ee.png)
+![image](https://user-images.githubusercontent.com/47582549/124770380-dd77a580-df6c-11eb-886a-659e0d41cb58.png)
+
+* Data models / API specifications of the core functionality
+CCS AI engine layer provide the core app layer，collect each chain value base on different Chain Feature （like cost，speed, etc) value and user Preference to use AI select best chain when upload files。Also Provide single&simple interface  that reduce complex for end user。Following are RestAPI for main entries for CCS AI engine.
+![image](https://user-images.githubusercontent.com/47582549/124771299-9938d500-df6d-11eb-954d-f393c4caf8fa.png)
+
+* Example data for strategies. Base on each chain & strategy feature to train AI mode for selecting chain against different requirement.
+![image](https://user-images.githubusercontent.com/47582549/124771382-aa81e180-df6d-11eb-9936-5763e0d22d18.png)
+
+* An overview of the technology stack to be used
+CCSUI Mainly use react,ipfs framework/technology stack.  For each 3rd chains, use each chain’s(like Crust,FileCoin,Swarm) API to interact(login, get CID, upload etc)
+CCS AI engine use nodejs,js technology to develop AI selector.
+![image](https://user-images.githubusercontent.com/47582549/124771461-b7063a00-df6d-11eb-8267-fa20fd41202e.png)
```

Comment:
The picture looks good, but the text part shows "CSS", is that a typo?

Reply:
thanks for your question
CCS in our context standards for "Cross Chain Storage" which is our project name, not a typo.

Reply:
I saw both CCS and CSS from https://user-images.githubusercontent.com/47582549/124771461-b7063a00-df6d-11eb-8267-fa20fd41202e.png

Reply:
Sorry, it's a wrong typing from our engineer